hr_period/tests/test_hr_fiscalyear.py

```diff
@@ -282,10 +293,14 @@ def test_create_periods_monthly_payment_last_day(self):
         fy.create_periods()
         periods = self.get_periods(fy)
         self.assertEqual(len(periods), 12)
-        self.assertEqual(periods[0].date_payment, '2015-01-31')
-        self.assertEqual(periods[1].date_payment, '2015-02-28')
-        self.assertEqual(periods[2].date_payment, '2015-03-31')
-        self.assertEqual(periods[11].date_payment, '2015-12-31')
+        self.assertEqual(periods[0].date_payment.strftime('%Y-%m-%d'),
+                         '2015-01-31')
+        self.assertEqual(periods[1].date_payment.strftime('%Y-%m-%d'),
```

Comment:
Maybe redo this to comparing to `date(2015, 2, 28)`?

Reply:
why?

Reply:
less copy-paste of `strftime` + format?

Reply:
done